modules/sharedaddy/sharing-sources.php

```diff
@@ -623,20 +623,28 @@ public function get_display( $post ) {
 		$via = $this->sharing_twitter_via( $post );
 
 		if ( $via ) {
-			$via = '&via=' . rawurlencode( $via );
+			$via = 'data-via="' . $via . '"';
 		} else {
 			$via = '';
 		}
 
 		$related = $this->get_related_accounts( $post );
 		if ( ! empty( $related ) && $related !== $via ) {
-			$via .= '&related=' . rawurlencode( $related );
+			$related = 'data-related="' . $related . '"';
+		} else {
+			$related = '';
 		}
 
 		if ( $this->smart ) {
 			$share_url = $this->get_share_url( $post->ID );
 			$post_title = $this->get_share_title( $post->ID );
-			return '<div class="twitter_button"><iframe allowtransparency="true" frameborder="0" scrolling="no" src="' . esc_url( $this->http() . '://platform.twitter.com/widgets/tweet_button.html?url=' . rawurlencode( $share_url ) . '&count=none&text=' . rawurlencode( $post_title . ':' ) . $via ) . '" style="width:55px; height:20px;"></iframe></div>';
+			return sprintf(
+				'<a href="https://twitter.com/share" class="twitter-share-button"{count} data-url="%1$s" data-text="%2$s" %3$s %4$s>Tweet</a>',
```

Comment:
I removed it in 7cdd7d83fb0d4e5a71783f46cf19a8558a4aae67. It was added in their generator, but that's probably not that useful now that sharing counts have been removed.